networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/subnets/AttestationTopicSubscriber.java

```diff
@@ -63,17 +63,21 @@ public synchronized void subscribeToPersistentSubnets(
     for (SubnetSubscription subnetSubscription : newSubscriptions) {
       int subnetId = subnetSubscription.getSubnetId();
       shouldUpdateENR = persistentSubnetIdSet.add(subnetId) || shouldUpdateENR;
-
-      UInt64 existingUnsubscriptionSlot =
-          subnetIdToUnsubscribeSlot.computeIfAbsent(
-              subnetId,
-              (key) -> {
-                eth2P2PNetwork.subscribeToAttestationSubnetId(subnetId);
-                return ZERO;
-              });
-
-      subnetIdToUnsubscribeSlot.put(
-          subnetId, existingUnsubscriptionSlot.max(subnetSubscription.getUnsubscriptionSlot()));
+      LOG.trace("Subscribing to persistent subnet {}", subnetId);
```

Comment:
let's add ` with unsubscribe due at slot {}` similar to aggregation committee log line

Reply:
Done